libpod/container_internal_linux.go

```diff
@@ -510,15 +550,43 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 	}
 
 	if !options.Keep {
-		// Remove log file
-		os.Remove(filepath.Join(c.bundlePath(), "dump.log"))
-		// Remove statistic file
-		os.Remove(filepath.Join(c.bundlePath(), "stats-dump"))
+		cleanup := [...]string{
```

Comment:
The reason for [...]string instead of []string, is that I thought [...] is an array and less overhead than a slice. But I am happy to change it to []string.